pubspec.yaml

```diff
@@ -1,37 +1,21 @@
 name: rectangle_detector
-description: A new flutter plugin project.
+description: Flutter plugin to detect edges of objects, scan paper, and detect rectangle. Using OpenCV for Android and and VisionKit IOS.
 version: 0.0.1
-author:
-homepage:
+homepage: https://github.com/oddbit/flutter_rectangle_detector
 
 environment:
   sdk: ">=2.7.0 <3.0.0"
   flutter: ">=1.10.0"
 
 dependencies:
   meta: ^1.0.4
-  platform: ^2.0.0
   flutter:
     sdk: flutter
 
 dev_dependencies:
-  test: ^1.3.0
-  mockito: ^3.0.0
   flutter_test:
     sdk: flutter
-  e2e: ^0.2.1
```

Comment:
@paramadharmika i didn't find these dependencies used anywhere. So i deleted it also. Please let me know if i'm wrong and these dependencies are used

Reply:
Alright lets delete that